<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scraper_panel</class>
 <widget class="QWidget" name="scraper_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1668</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_tasks">
       <property name="text">
        <string>Tasks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="taskTable">
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>URL</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_logs">
       <property name="text">
        <string>Logs</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="btnInfo">
         <property name="text">
          <string>INFO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnWarn">
         <property name="text">
          <string>WARN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnError">
         <property name="text">
          <string>ERROR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnClearLog">
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="logOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnAddTask">
       <property name="text">
        <string>Add Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
